### PR TITLE
New Extension: Divider

### PR DESCRIPTION
**Pull request by Wikiki**
New extension link added to documentation to easily display a horizontal or vertical divider

### CODE REVIEW

**Comment by Wikiki** [comment]
Sorry. My editor beautified the file.
It should be better now.

File: docs/extensions.html [hunk 1]
```diff
@@ -3,6 +3,11 @@
 layout: default
 route: extensions
 extensions:
+- name: bulma-divider
+  url: https://github.com/Wikiki/bulma-divider
+  slug: bulma-divider
+  width: 2224
+  height: 1352
```
Comment (jgthms) [inline docs/extensions.html]: These values should be divided by 2 for retina.

**Comment by Wikiki** [comment]
Divide image size by 2

**Comment by jgthms** [comment]
Conflict

**Pull request merged**